obspy/io/segy/segy.py

```diff
@@ -940,8 +939,8 @@ def _read_segy(file, endian=None, textual_header_encoding=None,
         Defaults to False.
     :type headonly: bool
     :param headonly: Determines whether or not the actual data records will be
-        read and unpacked. Has a huge impact on memory usage. Data can be read
-        and unpacked on-the-fly after reading the file. Defaults to False.
+        read and unpacked. Has a huge impact on memory usage. Data
+            will not be unpackable on-the-fly after reading the file. Defaults to False.
```

Comment:
CI is complaining because it's not on the same indentation as line above..

Reply:
That's what I get from changing it on Github. Sorry not doing that again.